codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttSubAckMessage.java

```diff
@@ -23,14 +23,21 @@
 
     public MqttSubAckMessage(
             MqttFixedHeader mqttFixedHeader,
-            MqttMessageIdVariableHeader variableHeader,
+            MqttMessageIdAndPropertiesVariableHeader variableHeader,
```

Comment:
We can't do this as it would be API breakage

Reply:
There's a constructor with the old signature besides this one - the code that uses the old API will use the old version, while the code that uses MQTT 5 features will use the new one.